src/routes/(admin)/questionbank/single-question/[action]/[id]/+page.svelte

```
@@ -112,109 +116,143 @@
-					{#each totalOptions as { id, key, value }, index (id)}
-						<div class="group flex flex-row gap-4">
-							<div class="bg-primary-foreground h-12 w-12 rounded-sm text-center">
-								<p class="flex h-full w-full items-center justify-center text-xl font-semibold">
-									{key}
-								</p>
-							</div>
-							<div class="flex w-full flex-col gap-2">
-								<div class="flex flex-row rounded-sm border-1 border-black">
-									<GripVertical class="my-auto h-full  rounded-sm bg-gray-100" />
-									<Input class=" border-0" name={key} bind:value={totalOptions[index].value} />
+						{#each totalOptions as { id, key, value }, index (id)}
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)